package.json

```diff
@@ -29,12 +29,16 @@
     "jest": "^29.7.0",
     "openai": "^4.38.5",
     "pdf-parse": "^1.1.1",
+    "pdf-parse-debugging-disabled": "^1.1.1",
+    "pdf2json": "^3.1.2",
     "redis": "^4.6.13",
     "typescript": "^5.4.5",
     "uuid": "^9.0.1",
     "wikibase-sdk": "^10.0.2"
   },
   "devDependencies": {
-    "concurrently": "^8.2.2"
+    "concurrently": "^8.2.2",
+    "ts-node": "^10.9.2",
```

Comment:
We could probbaly remove ts-node here since we use the native --watch mode for nodejs.

Reply:
Thanks. Fixed